src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -71,14 +72,28 @@ private void initUI() {
         panelMain.add(createSpreadsheetPanel(), c);
 
         c.gridy = 3;
-        panelMain.add(createWorkspacePanel(), c);
+        c.fill = GridBagConstraints.HORIZONTAL;
+        JLabel lblNoti = new JLabel("Notification");
+        panelMain.add(lblNoti, c);
 
         c.gridy = 4;
-        panelMain.add(createColumnPanel(), c);
+        panelMain.add(createWorkspacePanel(), c);
 
         c.gridy = 5;
+        c.fill = GridBagConstraints.NONE;
+        c.anchor = GridBagConstraints.LINE_START;
+        panelMain.add(createColumnPanel(), c);
+
+        c.gridx = 1;
+        JButton btnRefresh = new JButton("Refresh");
```

Comment:
Wrap the refresh button and the column panel into one panel. This way we can control their size without affecting other components